aspnetcore/mobile/native-mobile-backend/sample/TodoAPI/src/TodoAPI/TodoAPI.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
@scottaddie  as you can see there is a project file. But the snipets 500 check returns the error

> Error: 😵 Project missing. A project (and optionally a solution file) must be in this directory or one of the parent directories to validate and build this code.